modules/nodes-sigstore-configure.adoc

```diff
@@ -0,0 +1,66 @@
+
+.Example image policy with a conflicting image identity
+[source,yaml]
+----
+API Version:  config.openshift.io/v1alpha1
+Kind:         ImagePolicy
+Name:         p0
+Namespace:    mynamespace
+# ...
+Status:
+  Conditions:
+    Message: has conflicting scope(s) ["example.com/global/image"] that equal to or nest inside existing clusterimagepolicy, only policy from clusterimagepolicy scope(s) will be applied
+    Reason: ConflictScopes
+# ...
+----
+
+[id="nodes-sigstore-configure-parameters_{context}"]
+== About image policy parameters
+
+The following parameters apply to cluster and/or image policies. For information on using these parameters, see _Creating a cluster image policy CR_ and _About Creating an image policy CR_. 
+
+// Based on https://github.com/openshift/api/blob/master/config/v1alpha1/zz_generated.crd-manifests/0000_10_config-operator_01_imagepolicies-TechPreviewNoUpgrade.crd.yaml
+
+`scopes`:: Defines a list of  image identities assigned to a policy. You must list at least one image identity. An image identity can refer to any of the following identities:
++
+--
+* An individual image, by using a tag or digest, such as `example.com/namespace/image:latest`
+* A repository by omitting the tag or digest, such as `example.com`
+* A repository namespace, such as `example.com/namespace/`
+* A registry host, by specifying only the host name and port number or a wildcard expression starting with `\*.`, such as `*.example.com` 
+--
++
+If multiple  image identities match a single image, the policy for only the most specific image identity applies. 
++
+In a cluster image policy, make sure that the policy allows deployment of the {product-title} images in the `quay.io/openshift-release-dev/ocp-release` and `quay.io/openshift-release-dev/ocp-v4.0-art-dev` repositories. Images in these repositories are required for cluster operation.
+
+`policy`:: Contains configuration to allow images from the sources listed in `scopes` to be verified, and defines how images not matching the verification policy will be treated. You must configure a `rootOfTrust` and `signedIdentity`.
+* `rootOfTrust`: Specifies the root of trust for the policy. Configure for either a puclic key, optionally including a Rekor verification, or a Fulcio certification.
+** `publicKey`:  Indicates that the policy relies on a Sigstore public key. You must specify a base64-encoded PEM format public key. You can optionally include Rekor verification.
+** `FulcioCAWithReko`: Indicates that the policy is based on the Fulcio certification. You must specify the following parameters:
+*** A base64-encoded PEM format Fulcio CA
+*** An OpenID Connect (OIDC) issuer 
+*** The email of the Fulcio authentication configuration
+*** The Rekor verification
+* `signed identity`: Specifies the approach used in the verification process to verify the image in the signature and the actual image itself. You must specify one of the following parameters for the match policy:
```

Comment:
signedIdentity is an optional field, you must specify one of the following parameters for the match policy once signedIdentity is configured. Or something like this just to clarify it is optional.